fission/src/ui/panels/configuring/ConfigureSubsystemsPanel.tsx

```diff
@@ -0,0 +1,277 @@
+                        ? PreferencesSystem.getRobotPreferences(robot.assemblyName).motors.filter(x => {
+                              if (x.name) return x.name != driver.info?.name
+                              return false
+                          })
+                        : []
+
+                    removedMotor.push({
+                        name: driver.info?.name ?? "",
+                        maxVelocity: vel,
+                        maxForce: force,
+                    })
+
+                    PreferencesSystem.getRobotPreferences(robot.assemblyName).motors = removedMotor
+                }
+            }
+
+            PreferencesSystem.savePreferences()
+        },
+        [driver, robot.mechanism, robot.assemblyName]
+    )
+
+    return (
+        <>
+            <Box component={"div"} display={"flex"} justifyContent={"space-between"} alignItems={"center"} gap={"1rem"}>
+                <Stack direction={StackDirection.Vertical} spacing={8} justify="start">
+                    <Label size={LabelSize.Medium}>
+                        {driver instanceof WheelDriver ? "Drive" : driver.info?.name ?? "UnnamedMotor"}
+                    </Label>
+                    <Slider
+                        min={0.1}
+                        max={driverSwitch(driver, 80, 40, 80) as number}
+                        value={velocity}
+                        label="Max Velocity"
+                        format={{ minimumFractionDigits: 2, maximumFractionDigits: 2 }}
+                        onChange={(_, _velocity: number | number[]) => {
+                            setVelocity(_velocity as number)
+                            onChange(_velocity as number, force)
+                        }}
+                        step={0.01}
+                    />
+                    {PreferencesSystem.getGlobalPreference("SubsystemGravity") || driver instanceof WheelDriver ? (
+                        <Slider
+                            min={driverSwitch(driver, 100, 20, 0.1) as number}
+                            max={driverSwitch(driver, 800, 150, 15) as number}
+                            value={force}
+                            label={driverSwitch(driver, "Max Force", "Max Torque", "Max Acceleration") as string}
+                            format={{ minimumFractionDigits: 2, maximumFractionDigits: 2 }}
+                            onChange={(_, _force: number | number[]) => {
+                                setForce(_force as number)
+                                onChange(velocity, _force as number)
+                            }}
+                            step={0.01}
+                        />
+                    ) : driver instanceof HingeDriver ? (
+                        <Label>Select Realistic Gravity in Settings for torque config</Label>
+                    ) : (
+                        <Label>Select Realistic Gravity in Settings for force config</Label>
```

Comment:
I'm not sure I'm a fan of these labels. I agree it would be helpful to add some sort of description but, at least to me, it feels weird to 'Select (a) Realistic Gravity' when I don't have any sort of indication of what the units here are.

Reply:
I could take them out entirely and let users figure out that torque/force config appears when they enable realistic gravity.  That could be in the tutorial. I like enable wording better at least if we do keep it in there. Do you have other wording ideas?

Reply:
Is `Enable Realistic Gravity` a setting?

Reply:
Yep, in General Settings. It overrides force/torque though, so users can't modify that acceleration if Realistic Gravity isn't enabled. That's what the note is for.

Reply:
Got it, for now I think it's fine. We should probably add something later for people like me who didn't know about the `Realistic Gravity` option 🙃